src/UI/Implementation/Component/Listing/Renderer.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+/* Copyright (c) 2016 Timon Amstutz <timon.amstutz@ilub.unibe.ch> Extended GPL, see docs/LICENSE */
+
+
+namespace ILIAS\UI\Implementation\Component\Listing;
+
+use ILIAS\UI\Implementation\Render\AbstractComponentRenderer;
+use ILIAS\UI\Renderer as RendererInterface;
+use ILIAS\UI\Component;
+
+/**
+ * Class Renderer
+ * @package ILIAS\UI\Implementation\Component\Listing\Descriptive
+ */
+class Renderer extends AbstractComponentRenderer {
+	/**
+	 * @inheritdocs
+	 */
+	public function render(Component\Component $component, RendererInterface $default_renderer) {
+		/**
+		 * @var Component\Listing\Listing $component
+		 */
+		$this->checkComponent($component);
+
+		if ($component instanceof Component\Listing\descriptive) {
```

Comment:
`descriptive` => "Descriptive" (i think its correct either way, but looks odd to me).